### PR TITLE
Make library page use 'yesterday' by default unless .env provides an override

### LINKED ISSUE #329: Prep the front end for end-to-end systems test
### Description

Remove the guard on either side of May 2022 that limits the library page from selecting other dates.
Set the default start date for data to be "yesterday" (or "last week" or similar)
Optionally - if a default start date is provided via .env variable, set that to the initial start date (so we can continue to test using May 2022 data without always scrolling through 6 months of the date picker)

note that query parameters are already in place that allow a visitor to link to the library page at a specific date

### User Story

_No response_

### Architectural Decision Records (ADRs)

_No response_

### PR DESCRIPTION
**Pull request by sknep**
Closes #329 and prepares the frontend for end-to-end testing.

Removes the guard on either side of May 2022 that limits the library page from selecting other dates. The default start date for backend data is now "yesterday", so we can see recently posted presence data on first load on production.

For future development:

If a default start date is provided to the front end via .env variable (`VITE_DEFAULT_DATE_OVERRIDE=2022-05-31`), the Library page will load at that start date on first visit (so we can continue to test using May 2022 data without always scrolling through 6 months of the date picker). It's just for developer quality of life/testing.

### CODE REVIEW

File: imls-frontend/src/assets/index.scss [hunk 1]
```diff
@@ -28,4 +28,8 @@
   &:hover {
     @include u-bg('blue-warm-10');
   }
+}
+
+.maxw-date-picker{
+  max-width: 20em;
```
Comment (sknep) [inline imls-frontend/src/assets/index.scss:34]: I found a small visual bug on the datepicker and fixed it here

File: imls-frontend/src/views/PageLibrary.vue [hunk 2]
```diff
@@ -36,23 +43,28 @@ export default {
     }
   },
   computed: {
-    activeDate() {
-      return this.selectedDate;
+    selectedDateUTC() {
```
Comment (sknep) [inline imls-frontend/src/views/PageLibrary.vue:46]: I decided to make sure it was clear that these dates were in UTC (not ISO or some other format) so they've been renamed

**Review by bklaas — APPROVED** [review]
No red flags seen by this Vue-noob

**Review by anagradova — APPROVED** [review]

**Comment by anagradova** [comment]
LGTM

**Pull request merged**